_dashboards/management/management-index.md

```diff
@@ -20,7 +20,7 @@ Dashboards Management serves as the command center for customizing OpenSearch Da
 
 The following applications are available in Dashboards Management:
 
-- **Index Patterns:** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. An index pattern can point to one or more indexes, data streams, or index aliases. 
+- **[Index Patterns]({{site.url}}{{site.baseurl}}/dashboards/management/index-patterns/):** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. The Index Pattern tool gives you capabilities to create an index pattern from within the user interface. Index patterns point to one or more indexes, data streams, or index aliases. 
```

Comment:
NIT (and unrelated to these changes): IMO, it interesting the API for indexes is `indices` [[example]](https://opensearch.org/docs/latest/api-reference/cat/cat-indices/), but when we talk about it we use indexes. But I don't see enough people complaining about it.

Reply:
AWS, Microsoft, and Google all use "indexes" unless used in the context of a mathematical expression, so our style guide follows suit.